### PR TITLE
Rework docs

### PR DESCRIPTION
**Pull request by Eejit43**
This PR fully reworks the documentation, mainly adding example for how to use each feature

It also updates `md-release.yml` to account for the new file locations and formatting

### CODE REVIEW

**Comment by extremeheat** [comment]
Don't need to change the folder name

**Comment by Eejit43** [comment]
> Don't need to change the folder name

Well it isn't a singular _documentation_, it is _documentations_, plural, so it should be "docs"

**Comment by Eejit43** [comment]
> Don't need to change the folder name

would you prefer I revert that change?

**Comment by extremeheat** [comment]
Yes that should be reverted unless it's breaking something. Neither abbreviation is correct over the other

**Comment by Eejit43** [comment]
> Yes that should be reverted unless it's breaking something. Neither abbreviation is correct over the other

Will do :+1:

File: README.md [hunk 1]
```diff
@@ -1,34 +1,33 @@
...
-* find items by name
-* find block or item by name
-* find block or item by id
+This package allows the lookup of blocks, items, entities, etc. by name, id, etc., and for the easy lookup of other data.
 
 ## Example
 
 ```js
-const mcData=require("minecraft-data")("1.8.8")
+const MinecraftData = require('minecraft-data')
```
Comment (rom1504) [inline README.md:19]: Please undo this naming change

File: doc/api.md [hunk 2]
```diff
@@ -1,364 +1,675 @@
...
+
+or using es6 import syntax:
+
+```js
+import MinecraftData from 'minecraft-data'
+
+const minecraftData = MinecraftData('1.19')
+```
+
+All examples reference `MinecraftData` as the module, and `minecraftData` as the version data.
```
[the remark this answers is not in the document]
Comment (rom1504) [reply doc/api.md:19]: Variable names should start with lower case

[the remark this answers is not in the document]
Comment (rom1504) [reply doc/api.md:19]: Yes for example

File: doc/history.md [hunk 3]
```diff
@@ -1,682 +1,903 @@
...
+
+- update `minecraft-data`
 
 ## 2.110.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.109.0
-* update mcdata
```
Comment (rom1504) [inline doc/history.md:101]: This breaks the consistency with other PrismarineJs package and i don't see why make that change .

File: README.md [hunk 4]
```diff
@@ -1,34 +1,33 @@
...
-console.log(mcData.windows["minecraft:brewing_stand"])
-console.log(mcData.version)
-console.log(mcData.effectsByName["Haste"])
+console.log(minecraftData.blocksByName['stone']) // Information for "Stone"
+console.log(minecraftData.windows['minecraft:brewing_stand']) // Information for the "Brewing Stand" GUI
+console.log(minecraftData.version) // Information about the current version
+console.log(minecraftData.effectsByName['Haste']) // Information for the "Haste" effect
 ```
 
 ## Documentation
```
Comment (rom1504) [inline README.md:31]: We use * everywhere else, let's keep this

File: doc/api.md [hunk 5]
```diff
@@ -1,364 +1,675 @@
...
 
-### minecraft-data.enchantmentsByName
+Example:
 
-enchantments indexed by name
+```js
+console.log(minecraftData.enchantments[37]) // Object containing information for "Mending"
+```
+
+### `minecraftData.enchantmentsByName`
```
Comment (rom1504) [inline doc/api.md:290]: It looks better without the backtick and more consistent with other places, let's undo this.

File: doc/history.md (hunk 3 above)
Comment (Eejit43) [reply doc/history.md:101]: alright will do 👍

File: doc/api.md (hunk 5 above)
Comment (Eejit43) [reply doc/api.md]: Done 👍

File: doc/api.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Eejit43) [reply doc/api.md]: Done 👍

**Pull request merged**